<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>七只蜗牛</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>:/Resource/main/icon.ico</selectedon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #444444;
    color: #FFFFFF;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionReset"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionReference"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionMove"/>
    <addaction name="actionRotate"/>
    <addaction name="actionScale"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionScene_Explorer"/>
    <addaction name="actionLayer_Explorer"/>
   </widget>
   <widget class="QMenu" name="menuGroup">
    <property name="title">
     <string>Group</string>
    </property>
    <addaction name="actionGroup"/>
    <addaction name="actionUngroup"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_2"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionAttach"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionViewCube"/>
    <addaction name="actionXview"/>
   </widget>
   <widget class="QMenu" name="menuCreate">
    <property name="title">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuRendering">
    <property name="title">
     <string>Rendering</string>
    </property>
    <addaction name="actionRender"/>
   </widget>
   <widget class="QMenu" name="menuCustomize">
    <property name="title">
     <string>Customize</string>
    </property>
    <addaction name="actionHotkey_Editor"/>
    <addaction name="actionPreferences_2"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionWhat_s_New"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionWindows"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuGroup"/>
   <addaction name="menuWindows"/>
   <addaction name="menuView"/>
   <addaction name="menuCreate"/>
   <addaction name="menuRendering"/>
   <addaction name="menuCustomize"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMove"/>
   <addaction name="actionRotate"/>
   <addaction name="actionScale"/>
   <addaction name="separator"/>
   <addaction name="actionkeyboard"/>
   <addaction name="action2_5"/>
   <addaction name="actionangle"/>
   <addaction name="actionPercent"/>
   <addaction name="actionSpinner"/>
   <addaction name="separator"/>
   <addaction name="actionScene"/>
   <addaction name="actionLayer"/>
   <addaction name="separator"/>
   <addaction name="actionswitch"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_7">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSave">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/SaveFile_32.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/SaveAs_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionReference">
   <property name="text">
    <string>Reference</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/Preferences_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/Undo_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRedo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/Redo_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/Move_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/RotateCW_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/UniformScale_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
  </action>
  <action name="actionScene_Explorer">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/ToggleSceneExplorer_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Scene Explorer...</string>
   </property>
  </action>
  <action name="actionLayer_Explorer">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/ToggleLayerExplorer_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Layer Explorer...</string>
   </property>
  </action>
  <action name="actionGroup">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/Group_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Group...</string>
   </property>
  </action>
  <action name="actionUngroup">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/Ungroup_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Ungroup</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/Open_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/Close_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionAttach">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/Attach_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Attach</string>
   </property>
  </action>
  <action name="actionViewCube">
   <property name="text">
    <string>ViewCube</string>
   </property>
  </action>
  <action name="actionXview">
   <property name="text">
    <string>Xview</string>
   </property>
  </action>
  <action name="actionRender">
   <property name="text">
    <string>Render</string>
   </property>
  </action>
  <action name="actionHotkey_Editor">
   <property name="text">
    <string>Hotkey Editor...</string>
   </property>
  </action>
  <action name="actionPreferences_2">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/Preferences_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionWhat_s_New">
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/heart.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>What's New</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionkeyboard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/KeyboardOverrideToggle_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>keyboard</string>
   </property>
   <property name="toolTip">
    <string>keyboard</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action2_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/SnapsToggle2.5_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>2.5</string>
   </property>
   <property name="toolTip">
    <string>2.5</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/AngleSnapToggle_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>angle</string>
   </property>
   <property name="toolTip">
    <string>angle</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionPercent">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/PercentSnapToggle_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Percent</string>
   </property>
   <property name="toolTip">
    <string>Percent</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionSpinner">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/SpinnerSnapToggle_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Spinner</string>
   </property>
   <property name="toolTip">
    <string>Spinner</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionScene">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/ToggleSceneExplorer_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Scene</string>
   </property>
   <property name="toolTip">
    <string>Scene</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionLayer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/ToggleLayerExplorer_48.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>Layer</string>
   </property>
   <property name="toolTip">
    <string>Layer</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionswitch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <selectedon>:/Resource/main/heart.png</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>switch</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionWindows">
   <property name="text">
    <string>Windows</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
